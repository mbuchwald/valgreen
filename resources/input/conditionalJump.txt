==22== Conditional jump or move depends on uninitialised value(s)
==22==    at 0x4C32D39: __strlen_sse2 (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==22==    by 0x11D3DB: join (strutil.c:85)
==22==    by 0x115D1C: test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)
==22==  Uninitialised value was created by a heap allocation
==22==    at 0x4C2FB0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==22==    by 0x11D394: join (strutil.c:80)
==22==    by 0x115D1C: test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)
==22== 
==22== Conditional jump or move depends on uninitialised value(s)
==22==    at 0x4C32996: strcat (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==22==    by 0x11D426: join (strutil.c:96)
==22==    by 0x115D1C: test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)
==22== 
==22== Conditional jump or move depends on uninitialised value(s)
==22==    at 0x4C329D0: strcat (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==22==    by 0x11D45C: join (strutil.c:98)
==22==    by 0x115D1C: test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)
==22==  Uninitialised value was created by a stack allocation
==22==    at 0x11D348: join (strutil.c:64)
==22== 