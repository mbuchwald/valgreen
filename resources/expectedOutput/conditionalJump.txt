 [1m1-Conditional jump or move depends on uninitialised value(s)[0m
   at __strlen_sse2 [90m(in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)[39m
    [90m[31m[1mby join (strutil.c:85)[0m[39m[39m
    [90mby test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)[39m
   [1mUninitialised value was created by a heap allocation[0m
   at malloc [90m(in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)[39m
    [90m[32mby join (strutil.c:80)[39m[39m
    [90mby test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)[39m
    [90m[39m
 [1m2-Conditional jump or move depends on uninitialised value(s)[0m
   at strcat [90m(in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)[39m
    [90m[31m[1mby join (strutil.c:96)[0m[39m[39m
    [90mby test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)[39m
    [90m[39m
 [1m3-Conditional jump or move depends on uninitialised value(s)[0m
   at strcat [90m(in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)[39m
    [90m[31m[1mby join (strutil.c:98)[0m[39m[39m
    [90mby test_join_unir_dos_Test::TestBody() (strutil_test.cc:195)[39m
   [1mUninitialised value was created by a stack allocation[0m
   [32mat join (strutil.c:64)[39m
    [90m[39m
